testsuite/features/step_definitions/command_steps.rb

```diff
@@ -190,7 +190,11 @@
 
 When(/^vendor change should be enabled for "(?:[^"]*)" on "([^"]*)"$/) do |host|
   node = get_target(host)
-  _result, return_code = node.run("grep -- --allow-vendor-change /var/log/zypper.log")
+  pattern = '--allow-vendor-change'
+  log = '/var/log/zypper.log'
+  rotated_log = "#{log}-#{Time.now.strftime('%Y%m%d')}.xz"
+  _, return_code = node.run("grep -- #{pattern} #{log} || xzdec #{rotated_log} | grep -- #{pattern}")
```

Comment:
If this method should work for all our clients, do you know if xzdec and grep are installed by default in centos, ubuntu, sle, etc?

Reply:
@mseidl He must know if this is only for SLES.